pkg/generated/restapi/configure_fulcio_server.go

```diff
@@ -95,10 +98,29 @@ func configureAPI(api *operations.FulcioServerAPI) http.Handler {
 		if idToken == nil {
 			return nil, goaerrors.New(http.StatusUnauthorized, strings.Join(errs, ","))
 		}
+		if _, ok, err := oauthflow.EmailFromIDToken(idToken); !ok || err != nil {
```

Comment:
I think this will break SPIFFE support since it won't have an email in the token